.github/sync_labels.py

```diff
@@ -631,14 +632,16 @@ def gh_cmd(self, cmd, arg, option):
         issue = 'issue'
         if self._pr:
             issue = 'pr'
+        # workaround for gh bug, it cannot deduce repo from url automatically
```

Comment:
Maybe link https://github.com/cli/cli/issues/11055?

Reply:
reasonable, but then if they release a patch release ([soon](https://github.com/cli/cli/issues/10071#issuecomment-2934977180)) then this would be unnecessary anyway.

Reply:
Sure, but we probably have to wait a bit until the github image is using the new gh version (not sure how/when they update)